myportal/soap_service/server_med_rec.php

```diff
@@ -314,7 +315,7 @@ private function getIncudes($val)
         $copays = 0.00;
         foreach ($ar['newpatient'] as $be) {
             $ta = split(":", $be);
-            $billing = getPatientBillingEncounter($pid, $ta[1]);
+            $billing = BillingUtilities::getBillingByEncounter($pid, $ta[1]);
```

Comment:
TODO
revert the changes in this script since the function getPatientBillingEncounter() is in report.inc and also check if any billing functions are being called in this script